### PR TITLE
[COM-1348] Updates icons and exports Autocomplete and Highlight components

### PR DESCRIPTION
**Pull request by wescosta**
**Checklist**

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

**Please check if the PR fulfills these requirements**

- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Storybook or docs have been added / updated (for bug fixes / features)

**What is the ticket / issue associated with this change?** (Link to the ticket or issue)

**Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

**Other information**:

**Storybook Link** (Link to a local storybook url to review new functionality)

After

https://5eed577398d08d0022b52d76-qyzzkerhev.chromatic.com/?path=/story/components-form-autocomplete--all

Before

https://storybook.cmpsr.io/?path=/story/components-form-autocomplete--all

### CODE REVIEW

File: packages/components/src/components/form/Autocomplete/Autocomplete.tsx [hunk 1]
```diff
@@ -16,6 +16,21 @@ import {
...
+      // this prevents the menu from opening when there is no input value
+      return {
+        ...changes, // default Downshift new state changes on item selection.
+        isOpen: Boolean(changes.inputValue), // keep the menu open if there is a value.
+      };
+    default:
+      return changes; // otherwise business as usual.
+  }
+};
+
```
Comment (guilean) [inline packages/components/src/components/form/Autocomplete/Autocomplete.tsx:33]: @wescosta was this required by product/design?

Comment (wescosta) [reply packages/components/src/components/form/Autocomplete/Autocomplete.tsx:33]: Yes. As per @rob-w-tyrrell request, we want to show the autocomplete suggestion only when the person has typed at least 1 character.

Comment (guilean) [reply packages/components/src/components/form/Autocomplete/Autocomplete.tsx:33]: Ok, design had a different opinion about that when the component was implemented. Not sure if we want to add a property to control that behaviour

Comment (wescosta) [reply packages/components/src/components/form/Autocomplete/Autocomplete.tsx:33]: Ok. We can check with Rob and Hans tomorrow.

Comment (wescosta) [reply packages/components/src/components/form/Autocomplete/Autocomplete.tsx]: [added an optional expectedCharLengthToOpen prop to Autocomplete component](https://github.com/cmpsr/composer/pull/785/commits/05f886bdf4f1d0540f754ac4c096f994f898de22). Thus we can keep the original behavior while allowing to only show results when the user has typed something.

**Review by guilean — COMMENTED** [review]
Should we add a story file for `Highlight`?

**Review by guilean — APPROVED** [review]

**Pull request merged**